frontend/tailwind.config.js

```diff
@@ -0,0 +1,50 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+	darkMode: ["class"],
```

Comment:
this file is using tabs unlike all others where spaces are used
it would be great to align it with the rest of the project